.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,20 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📃 User documentation
+    url: https://support.signal.org/
+    about: Search for any questions you have about Signal
+  - name: 📖 Developer documentation
+    url: https://signal.org/docs/
+    about: Official Signal developer documentation 
+  - name: 💬 Community support
+    url: https://community.signalusers.org/c/support/
+    about: Feel free to ask anything
+  - name: ✨ Feature request
+    url: https://community.signalusers.org/c/feature-requests/
```

Should this be something like:
```suggestion
    url: https://community.signalusers.org/c/feature-requests/android-feature-requests/18
```
To bring them directly to the android tag within that topic.

This is great by the way!